### PR TITLE
Redirecting AIRO to airo

### PR DESCRIPTION
**Pull request by DelaramGlp**

### CODE REVIEW

**Comment by dgarijo** [comment]
@DelaramGlp, please do not delete yourself as the maintainer, just in case future updates are requested.

**Comment by dgarijo** [comment]
Sorry, now you seem to have generated a different PR. Can you please revise this one?

File: AIRO/README.md [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
+# AIRO (AI Risk Ontology)
 
-**Contact:**
-This ontology is created and maintained by:
-Delaram Golpayegani
-PhD Researcher
-ADAPT Centre, Trinity College Dublin, Dublin, Ireland
-delaram.golpayegani@adaptcentre.ie
-GitHub: delaramglp
+This directory is moved to https://github.com/perma-id/w3id.org/tree/master/airo.
```
Comment (TallTed) [inline AIRO/README.md:4]:
```suggestion
This directory is moved to https://github.com/perma-id/w3id.org/tree/master/airo. 

## Contact
@delaramglp
```

**Comment by davidlehn** [comment]
Closing as `AIRO` has been removed in favor of a special-case top-level redirect to `airo`.

**Pull request closed without merging**